webrtc.html

```diff
@@ -5536,6 +5538,16 @@ <h2>Attributes</h2>
               <p>On getting, the attribute MUST return the value of the
               <a>[[\SenderRtcpTransport]]</a> slot.</p>
             </dd>
+            <dt><dfn><code>msid</code></dfn> of type <span class=
+            "idlAttrType"><a>DOMString</a></span>, readonly</dt>
+            <dd>
+              <p>msid value of the related SDP mline.</p>
```

Comment:
What do you mean by "msid value" here? Both fields, or just one of them?
(note - the track part used to be available as track.id, but after replaceTrack(), the relationship broke, so making it available makes sense.)

Reply:
I mean the MediaStreamTrack id.
Updating it ;)